modules/prebidServerBidAdapter/index.js

```diff
@@ -89,7 +89,7 @@ config.setDefaults({
  * @return {boolean}
  */
 function updateConfigDefaultVendor(option) {
-  if (option.defaultVendor) {
+  if (option.defaultVendor && option.enabled !== false) {
```

Comment:
If s2sConfig does not explicitly state enabled === true then it always defaults to false (even though defaultVendor json option has it to true).
It is being overwritten before this line by the defaultS2sConfig here: https://github.com/prebid/Prebid.js/blob/master/modules/prebidServerBidAdapter/index.js#L75

I should have caught this during my PR review, sorry @patmmccann

Reply:
You are saying if someone selects defaultVendor and sets enabled = false the override is still occuring? I thought https://github.com/prebid/Prebid.js/blob/87ff2b8d0ba030c54b19056ccc5227b4baa1a916/modules/prebidServerBidAdapter/index.js#L100 was the line that did the overwrite and this change would prevent that from executing

Reply:
No, the opposite actually.

Now it is ignoring that defaultVendor.enabled flag.

So if you have this:

```js
{
   defaultVendor: 'rubicon',
   bidders: ['rubicon']
}
```

It will have `enabled: false`

So you have to explicitly do:

```js
{
   enabled: true,
   defaultVendor: 'rubicon',
   bidders: ['rubicon']
}
```


I have not pinpointed exact spot yet since I am busy on something else, but I should be able to add a PR to address later this week.